mkdocs/pages/en/devbook/transactions/messages.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Messages
+---
+
+# Sending Messages with Transfer Transactions
+
+<Transfer transactions:> can include an optional message field, which allows attaching up to 1,024 bytes of data to the
+transaction.
+Messages can be sent as plain text or encrypted using the recipient's public key, ensuring only the intended recipient
+can read them.
+
+This tutorial shows how to send both plain and encrypted messages, and demonstrates how to decrypt received messages.
+
+## Prerequisites
+
+If you have not done so already, start with the [Hello World](../start/hello-world.md) tutorial to make sure your
+development environment is set up correctly.
+
+Additionally, check the [Transfer transaction](transfer.md) tutorial to understand how fee
```

Comment:
```suggestion
Additionally, check the [Transfer transaction](./transfer.md) tutorial to understand how fee
```

It's not critical, but in the past I've used a plugin that automatically searched for files if they had not path.
I'm not using it now, but just in case I'll ever bring it back, always include the path in file names, even if it's just `./`